models/comments-model.js

```diff
@@ -0,0 +1,18 @@
+const db = require("../db/connection.js")
+
+function fetchCommentByArticleId(articleId){
+    return db.query(
+        `SELECT comments.* 
+        FROM comments INNER JOIN articles 
+        ON comments.article_id = articles.article_id
```

Comment:
why are you joining articles here ? Could you not just select all from comments where comments.article_id = article_id